docs/design/autoware-interfaces/components/perception.md

```diff
@@ -0,0 +1,35 @@
+# Perception
+
+![Node diagram](images/Perception-Bus-ODD-Architecture.drawio.svg)
+
+## Inputs
+
+### PointCloud
+
+PointCloud data published by Lidar.
+
+- [sensor_msgs/PointCloud2](http://docs.ros.org/en/api/sensor_msgs/html/msg/PointCloud2.html)
+
+### Image
+
+Image frame captured by camera.
+
+- [sensor_msgs/Image](http://docs.ros.org/en/api/sensor_msgs/html/msg/Image.html)
+
+### Vehicle kinematic state
+
+current position of ego, used in traffic signals recognization. See Iutputs of Planning.
+
+### Lanelet2 Map
+
+map of the environment. See Iutputs of Planning.
+
+## Output
+
+### 3D Object Predictions
```

Comment:
```suggestion
### 3D Dynamic Object Predictions
```
Items that were not present when the point cloud map was created are treated as Dynamic Objects. This means that objects such as cones under construction are included.